src/jit/gentree.h

```diff
@@ -991,6 +991,9 @@ struct GenTree
 #define GTF_RELOP_ZTT               0x08000000 // GT_<relop> -- Loop test cloned for converting while-loops into do-while
                                                //               with explicit "loop test" in the header block.
 
+#define GTF_JCMP_EQ                 0x80000000 // GTF_JCMP_EQ  -- Branch on equalrather than not equal
+#define GTF_JCMP_TST                0x40000000 // GTF_JCMP_TST -- Use test rather than compare
```

Comment:
`use test rather than compare` => `use test against zero instruction instead of compare against immediate`

Reply:
Done